routing/localchans/manager.go

```diff
@@ -180,19 +244,95 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 	return failedUpdates, nil
 }
 
+// createEdge recreates an edge and policy from an open channel in-memory.
+func (r *Manager) createEdge(channel *channeldb.OpenChannel,
+	timestamp time.Time) (*models.ChannelEdgeInfo,
```

Comment:
`timestamp` can be removed since it's always `time.Now`

Reply:
This is here because the timestamp is checked in a test. I like to feed in the timestamp in the test, so you don't rely on 'current time' in the test itself. Let me know whether that's ok.

Reply:
cool yeah makes sense.